views/window/window.html

```diff
@@ -1,4 +1,4 @@
-<div class="js-window modal d-flex c-window o-window" ng-class="{'active': active}">
+<div class="js-window modal d-flex c-window o-window u-user-select-none" ng-class="{'active': active}">
```

Comment:
dont add this to window. all windows wont be selectable. I prefer everything to be selectable and some stuff not, rather than nothing be selectable and some stuff is.

Reply:
If it is simpler to define everything as unselectable and then undo it for the few items that should be, why is that not the better option?

Reply:
@wallw-bits That was the original plan. I have gotten push back on changing CSS.

I'd like to know if I should continue adding u-user-select-none to individuals divs, or if I should declare rules within CSS. 

If you would prefer I do the CSS rules, let me know of a few areas to go off of. (Add layer description etc)

Reply:
Because in normal webpages everything is selectable by default. Its abnormal to have something not selectable. Its better to just make things you dont want selectable, not selectable. So you dont have users questioning why something isnt selectable

I dont even see why this ticket is even needed. I see no problem with the users selecting the text Areas on the areas tab. If the reason is when they are shift clicking on the map or something and stuff gets selected, that issue should be addressed instead of the bandaid fix of just making everything unselectable

Reply:
How about when the map is moused over, call
window.getSelection().removeAllRanges()

This will remove selected text

Reply:
That's no good because I could have a legitimate selection which that function would remove.

Reply:
It would only remove if you are mousing over the map, at which you've already left the area you are trying to select. It does fix the issue of trying to select some text and all the text gets selected which is not expected. There might be a better fix too, I think thats where the effort should be applied

Just because there is a big map in the background, doesn't mean it gets to break rules of the internet

Just checkout google maps https://www.google.com/maps/place/Washington,+DC/@38.8935755,-77.0846151,12z/data=!3m1!4b1!4m5!3m4!1s0x89b7c6de5af6e45b:0xc2524522d4885d2a!8m2!3d38.9071923!4d-77.0368707
They have no problem with allowing selection while having a big map in the background

Reply:
Google Maps doesn't have a drop target for text though - which is what makes the text selection a nuisance.